tmt/steps/__init__.py

```diff
@@ -242,21 +363,55 @@ def wake(self) -> None:
         # order to cover a very frequent use case 'tmt run --last report'
         # FIXME find a better way how to enable always-force per plugin
         if (isinstance(self, tmt.steps.report.Report) and
-                self.data[0].get('how') in ['display', 'html']):
+                self.data[0].how in ['display', 'html']):
             self.debug("Report step always force mode enabled.")
             self._workdir_cleanup()
             self.status('todo')
 
         # Nothing more to do when the step is already done
         if self.status() == 'done':
+            self.debug('step is done, not touching its data')
```

Comment:
```suggestion
            self.debug('Step is done, not touching its data.')
```

Reply:
Ack, will fix.

Reply:
Fixed in [8d096fd](https://github.com/teemtee/tmt/pull/1461/commits/8d096fd6dedfb1f4dab9cea49f75946075af5836)